src/dra/5298C.c

```diff
@@ -450,7 +450,42 @@ bool ScissorSprite(SPRT* sprite, MenuContext* context) {
     return false;
 }
 
-INCLUDE_ASM("dra/nonmatchings/5298C", func_800F5904);
+void func_800F5904(MenuContext* ctx, s32 x, s32 y, s32 w, u32 h, s32 u, s32 v,
+                   s32 idx, s32 unk2, bool disableTexShade, s32 unk4) {
+    SPRT* sp;
+    s32 otIdx;
+    u32* ot;
+    ot = g_CurrentBuffer->ot;
+    sp = &g_CurrentBuffer->sprite[g_GpuUsage.sp];
+    if (D_8013784C == 1) {
+        idx = 0x15E;
+    }
+    if (D_8013784C == 2) {
+        idx = 0x15D;
+    }
+    if (ctx != 0) {
+        otIdx = ctx->unk18 + 2;
+    } else {
+        otIdx = 0x1FF;
+    }
+    SetSemiTrans(sp, 0);
+    SetShadeTex(sp, disableTexShade);
+    sp->x0 = (s16)x;
+    sp->y0 = (s16)y;
+    sp->w = (s16)w;
+    sp->h = (s16)h;
+    sp->u0 = (u8)u;
+    sp->v0 = (u8)v;
+    if ((ctx == 0) || (ScissorSprite(sp, ctx) == false)) {
```

Comment:
Probably best to say ctx == NULL instead of 0?

Reply:
Updated it with NULL instead of 0 👍🏻